javascript/atoms/dom.js

```diff
@@ -312,18 +313,18 @@ bot.dom.getAttribute = function(element, attributeName) {
 /**
  * List of elements that support the "disabled" attribute, as defined by the
  * HTML 4.01 specification.
- * @private {!Array.<goog.dom.TagName>}
+ * @private {!Object<string, boolean>}
  * @const
  * @see http://www.w3.org/TR/html401/interact/forms.html#h-17.12.1
  */
-bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = [
+bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = goog.object.createSet([
   goog.dom.TagName.BUTTON,
```

Comment:
I feel like this is potentially relying on something too deep in the ecma definition... is there any browser quirk there? Also, when I dug deep in there, it seems it would actually attempt to call valueOf first and then toString...
http://www.ecma-international.org/ecma-262/6.0/#sec-toprimitive

Guess I'd prefer to be more explicit rather than potentially leaving ourselves open to a browser quirk